fonts/scripts/generate_angle_svgs.py

```
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
```

Ruff (0.14.4)

Shebang is present but file is not executable

(EXE001)